### ISSUE TITLE
NFL Boxscore not separating boxscore.away_players from boxscore.home_players

### ISSUE THREAD

**Issue by mdekarske** [roclark/sportsreference#501]
**Describe the bug**
NFL Boxscore not separating boxscore.away_players from boxscore.home_players

**To Reproduce**
Sample code which causes an issue.
```
from sportsreference.nfl.boxscore import Boxscores
from sportsreference.nfl.schedule import Boxscore
for schedweek in range(1,5):
    games_today = Boxscores(schedweek,2020)
    for week in games_today.games:
        for game in games_today.games[week]:
            bscore = Boxscore(game['boxscore'])
            if len(bscore.away_players) == 0 or len(bscore.home_players) == 0:
                print(f'{game["boxscore"]} Away {len(bscore.away_players)} Home {len(bscore.home_players)}')
```

**Expected behavior**
The players on the home team should show up in boxscore.home_players

**Screenshots**
Here are the games that there is a problem through week 4

202009130rav Away 69 Home 0
202009130ram Away 59 Home 0
202009200clt Away 64 Home 0
202009200oti Away 63 Home 0
202009200crd Away 61 Home 0
202009200sdg Away 59 Home 0
202009200htx Away 63 Home 0
202009210rai Away 62 Home 0
202009270sdg Away 54 Home 0
202009270clt Away 65 Home 0
202009270crd Away 61 Home 0
202009280rav Away 63 Home 0
202010040htx Away 64 Home 0
202010040ram Away 65 Home 0
202010040rai Away 65 Home 0

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Sportsreference Version 0.5.2

**Comment by mdekarske** [comment]
Using 202009200clt as an example.  home_abbreviation gets the abbrev from https://www.pro-football-reference.com/teams/clt/2020.htm which is CLT.  Then the team on the individual stats are listed as IND.

The Away/Home decision is if the home abbrev matches the players abbrev then it is home, else away.

**Comment by bmorgan0921** [comment]
For a quick, unofficial fix, one can insert the following code into the _boxscore.py_ file **between** lines **[584](https://github.com/roclark/sportsreference/blob/b77c5368bf04a20bcf43e3279236e635881dd9ee/sportsreference/nfl/boxscore.py#L584)** and **[585](https://github.com/roclark/sportsreference/blob/b77c5368bf04a20bcf43e3279236e635881dd9ee/sportsreference/nfl/boxscore.py#L585)**. Keep in mind that this is probably only a quick fix for one to keep working. It checks the name found and replaces it, if needed, before the conditional ``` if ``` statement that checks if home abbreviation matches the team abbreviation found on the player.

```
abbreviation_fix = {
            'BAL':'RAV', 'LAR':'RAM', 'IND':'CLT', 
            'TEN':'OTI', 'ARI':'CRD', 'LAC':'SDG', 
            'HOU':'HTX', 'LVR':'RAI'}
        try:
            name = abbreviation_fix[name]
        except:
            pass
```

I imagine that if the Washington Football Team (note: not addressed above) announces their new team name and abbreviation tomorrow, the above would need to be adjusted.

**Comment by mdekarske** [comment]
That works for the current year 2020!  It won't work for other years when the Raiders were referenced with OAK.[https://www.pro-football-reference.com/boxscores/201909090rai.htm](url)

I'm wondering if the fact that first team listed is always the away team.

**Comment by roclark** [comment]
Thank you both for the feedback! You are both spot-on about the abbreviations being the cause of the issue here, and I agree that it will continue to be an issue as various teams change their names and/or move. I found a fairly quick fix to this and will push an update soon once all of the tests pass. Essentially, I do a secondary pull of the abbreviations shown on the actual page (as opposed to just the URIs) and compare those abbreviations with the ones listed in the tables. In my testing, this works both for the code that @mdekarske provided above, as well as previous seasons that I tested and the correct number of players are returned for both home and away.

Thanks again! I will get an update to resolve this out shortly.
